crates/fluvio-connector-package/src/secret.rs

```diff
@@ -108,6 +151,12 @@ impl<T: AsRef<Path>> From<T> for FileSecretStore {
     }
 }
 
+fn get_or_init_secrets_re() -> Result<&'static Regex> {
```

Comment:
need unit tests for this

Reply:
addded a test for this in https://github.com/infinyon/fluvio/pull/3138/commits/cda51fa1e06684af9fcc0ddf2d7c0b622d0225d5